requirements/pip.in

```diff
@@ -0,0 +1,110 @@
+# Base packages
+pip
+virtualenv
+
+django~=3.2.0
+django-extensions
+django-polymorphic
+django-autoslug
+django-simple-history
```

Comment:
This used to be pinned at 3.0.0 and is now 3.1.1

3.1.0 has a lot of changes: https://django-simple-history.readthedocs.io/en/latest/#section-2

Reply:
We can kept it pinned in this PR to avoid this problem and solve it in a different PR 👍🏼